dev-docs/modules/symitriDapRtdProvider.md

```diff
@@ -38,14 +38,17 @@ pbjs.setConfig({
     auctionDelay: 2000,
     dataProviders: [
       {
-        name: "dap",
+        name: "symitriDap",
```

Comment:
@jpalladino84 -- can they use symitriDap in all versions of Prebid.js? Or is there a certain version that requires the older name?